### PR TITLE
Switch the default theme to the EXT Access theme

### PR DESCRIPTION
**Pull request by iwillig**
This should fix most of the current issues that where blocking us from using the access theme.  Some of the fixes are also addressed in a commit to GXP. 

Address #454

### CODE REVIEW

File: templates/geonode/geo_header.html [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {% load staticfiles %}
 {% load mapstory_tags %}
 
+
+<link rel="stylesheet"
+      type="text/css"
+      href="{% geonode_static "externals/ext/resources/css/xtheme-access.css"%}"/>
+
```
Comment (bartvde) [inline templates/geonode/geo_header.html:8]: why is this needed here as well? Should only be need in ext_header or not?

File: templates/upload/layer_upload_time.html [hunk 2]
```diff
@@ -67,6 +67,11 @@
         margin: 15px 0 5px;
     }
 </style>
+
+<link rel="stylesheet"
+      type="text/css"
+      href="{% geonode_static "externals/ext/resources/css/xtheme-access.css"%}"/>
+
```
Comment (bartvde) [inline templates/upload/layer_upload_time.html:74]: again, wondering why we need it here as well. Can you explain?

**Comment by iwillig** [comment]
No, 

Since we use Ext in a lot of different static pages we need to include the theme in those pages that don't extend ext_header.

**Comment by ischneider** [comment]
by not using ext_header unless needed, page loads are much faster

**Comment by bartvde** [comment]
I get that, but why would such pages need ext css then?

**Comment by iwillig** [comment]
I found that almost all of the pages that need the ext.js css (ie they have ext widgets on them) extend geo_header.html and not ext_header.html. Or something like that let me look into it  more.

**Comment by bartvde** [comment]
you can't have ext widgets without the ext scripts, so I am not getting how you can get into that situation without ext_header

**Comment by iwillig** [comment]
Thats a very good question Bart.

Ivan Willig

**Comment by iwillig** [comment]
Bart, 

It looks like the page_layout.html template pulls in the ext.js framework,

https://github.com/opengeo/mapstory/blob/master/templates/page_layout.html#L27-L28

So any page that extends that template is able to use ext.js widgets. Maybe the theme should be included there? 

Not sure.

**Comment by bartvde** [comment]
ah right, any reason why that template does not just include ext_header instead of defining its content itself?

**Comment by iwillig** [comment]
Not sure, but this is something I would be willing to look into.

**Comment by ischneider** [comment]
squashed, modified and merged with 9b8211599583f3e060cadbf52e91151cf355e2d0

i remove the css from the upload and geo header. they were conflicting with other styles and making text disappear
